luminance/src/metagl.rs

```diff
@@ -239,6 +239,8 @@ mod meta {
     pub const TRIANGLE_FAN: GLenum = 6;
     pub const TRIANGLE_STRIP: GLenum = 5;
     pub const TRIANGLES: GLenum = 4;
+    pub const PATCHES: GLenum = 14;
+    pub const PATCH_VERTICES: GLenum = 36466;
```

Comment:
Note to self: reorder these. I didn't notice that they were in alphabetical order before.

Reply:
Turns out they weren't strictly sorted before, either.

Reply:
Yeah you don’t have to mind that too much. Are you using the `no_std` part btw? I might change that a lot (I’ve never been able to run a binary using it… :disappointed: )

Reply:
Nope, I didn't even try to compile it, I just threw them in there for completeness' sake.

Reply:
Yeah, it’s ~fine to me. I might even temporarily disable them on `luminance-1.0` on re-introduce them later. it was an experiment that was too early.